app/api/events.py

```diff
@@ -210,15 +278,33 @@ def before_get_object(self, view_kwargs):
                 else:
                     view_kwargs['id'] = None
 
-    decorators = (is_coorganizer, )
+    def after_get(self, result):
+        result = changeToTimezone(result)
```

Comment:
@niranjan94 @shubham-padia 
I have kept this to jwt_required and not to `is_coorganizer`.
```
class EventDetail(ResourceDetail):
~~~ <snip>~~~~
decorators = (jwt_required,)
```
As `is_coorganizer` can only be used when [**Best way to implement permission system for APIs**( issue#3787) ](https://github.com/fossasia/open-event-orga-server/issues/3787)  is resolved.